test_conformance/SVM/test_unified_svm_consistency.cpp

```diff
@@ -0,0 +1,159 @@
+//
+// Copyright (c) 2020 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "common.h"
+#include <cinttypes>
+
+REGISTER_TEST(unified_svm_consistency)
+{
+    if (!is_extension_available(deviceID, "cl_khr_unified_svm"))
```

Comment:
I've not been following the unified svm test plan closely, but is the idea that the extension tests will live in `test_conformance/SVM` rather than `test_conformance/extensions/cl_khr_unified_svm`?

Reply:
Good question.  I don't think we've discussed this.  I'll create a new test executable if that's what the group prefers, but I think we've generally tried to minimize test executables, so I'm inclined to put the unified SVM tests here unless we have a good reason to do otherwise.